### PR TITLE
Updates tests and docstrings for cirq utils

### LINKED ISSUE #1124: Some functions in cirq_utils are not tested
This function:
https://github.com/unitaryfund/mitiq/blob/c818e44858b401b45f1f09e021c49ba24c5dd4a1/mitiq/interface/mitiq_cirq/cirq_utils.py#L37

and this function:
https://github.com/unitaryfund/mitiq/blob/c818e44858b401b45f1f09e021c49ba24c5dd4a1/mitiq/interface/mitiq_cirq/cirq_utils.py#L56

are not documented and tested.

Closing this issue requires:
- [x] Adding docstrings to both functions
- [x] Adding unit tests in this file: https://github.com/unitaryfund/mitiq/blob/master/mitiq/interface/mitiq_cirq/tests/test_cirq_utils.py

### PR DESCRIPTION
**Pull request by purva-thakre**
Description
-----------
Fixes #1124 

Checklist
---------

Check off the following once complete (or if not applicable) after opening the PR. The PR will be reviewed once this checklist is complete and all tests are passing.

- [x] I added unit tests for new code.
- [x] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [x] I used [Google-style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
- [x] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.
- [x] Added myself / the copyright holder to the AUTHORS file

If some items remain, you can mark this a [draft pull request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).

License
-------

- [x] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

Tips
----

- If the validation check fails:

    1. Run `make check-types` (from the root directory of the repository) and fix any [mypy](https://mypy.readthedocs.io/en/stable/) errors.

    2. Run `make check-style` and fix any [flake8](http://flake8.pycqa.org) errors.

    3. Run `make format` to format your code with the [black](https://black.readthedocs.io/en/stable/index.html) autoformatter.

  For more information, check the [Mitiq style guidelines](https://mitiq.readthedocs.io/en/stable/contributing.html#style-guidelines).

- Write "Fixes #XYZ" in the description if this PR fixes Issue #XYZ.

### CODE REVIEW

**Comment by purva-thakre** [comment]
@andreamari This is ready for review

**Comment by andreamari** [comment]
Sorry for the delay @purva-thakre, I hope to review this soon.

**Review by andreamari — APPROVED** [review]
LGTM!  I left just a few small suggestions.

Thanks @purva-thakre !

File: mitiq/interface/mitiq_cirq/cirq_utils.py [hunk 1]
```diff
@@ -29,6 +29,19 @@ def sample_bitstrings(
     sampler: cirq.Sampler = cirq.DensityMatrixSimulator(),
     shots: int = 8192,
 ) -> MeasurementResult:
+    """Adds noise to the input circuit. The noise is added based on a particular
+    noise model and some value for the error rate.
```
Comment (andreamari) [inline mitiq/interface/mitiq_cirq/cirq_utils.py:33]:
```suggestion
    """Adds noise to the input circuit. The noise is added based on a
    particular noise model and some value for the error rate.
```

**Pull request merged**